### PR TITLE
Refactor tina fields

### PR DESCRIPTION
**Pull request by jeffsee55**
WIP - move stuff

### CODE REVIEW

File: packages/@tinacms/fields/src/components/TextField.ts [hunk 1]
```diff
@@ -29,6 +29,6 @@ export interface TextFieldProps extends a {
   ref?: any
 }
 
-export const TextField = styled.input<TextFieldProps>`
+export const BaseTextField = styled.input<TextFieldProps>`
```
Comment (ncphillips) [inline packages/@tinacms/fields/src/components/TextField.ts]: I don't think this should have been changed

Comment (jeffsee55) [reply packages/@tinacms/fields/src/components/TextField.ts]: The `TextFieldPlugin` module exports `TextField` as well. This is just a WIP but there are a few of these types of collisions that I need to think about

Comment (ncphillips) [reply packages/@tinacms/fields/src/components/TextField.ts]: oof 🤔

File: packages/@tinacms/fields/src/plugins/BlocksFieldPlugin.tsx [hunk 2]
```diff
@@ -570,7 +570,7 @@ const Panel = function Panel({
   )
 }
 
-export default {
+export const BlocksFieldPlugin = {
```
Comment (jeffsee55) [inline packages/@tinacms/fields/src/plugins/BlocksFieldPlugin.tsx]: 👍

**Pull request merged**